src/opera/executor/ansible.py

```diff
@@ -27,7 +27,7 @@ def _get_inventory(host):
     return yaml.safe_dump(dict(all=dict(hosts=dict(opera=inventory))))
 
 
-def run(host, primary, dependencies, vars):
+def run(host, primary, dependencies, vars, verbose=False):
```

Comment:
Is there a reason behind the default value here? If we are carrying this parameter over several layers of the app, it would be wiser to make it required to ensure we did not forget to parameterize any of the places that need verbosity adjusted.

Reply:
Yeah, I agree. So I have removed default False values for verbose function parameter so this should make it to be required.